app/assets/javascripts/admin/addon/controllers/admin-backups-index.js

```diff
@@ -15,6 +15,11 @@ export default Controller.extend({
   enableExperimentalBackupUploader: setting(
     "enable_experimental_backup_uploader"
   ),
+  enableDirectS3Uploads: setting("enable_direct_s3_uploads"),
+  enableExperimentalS3BackupUploader: and(
```

Comment:
It's simpler to use the `(and` helper in templates these days than creating new computed properties. You can also remove `enableDirectS3Uploads`.